tests/acl/test_acl.py

```diff
@@ -175,13 +175,13 @@ def get_t2_info(duthosts, tbinfo):
                     acl_table_ports[namespace].append(port)
```

Comment:
can we check if namespace == `DEFAULT_NAMESPACE`  to add port to `acl_table_ports[' ']` else add to `acl_table_ports[namespace]`  to avoid causing issues for multi asic platforms

Reply:
What's the reason we don't want to run the `config acl add table` command in the global namespace on multi-asic duts?
If we don't do this then when we install the ACL rules `config acl update full` in the global namespace it will fail because the ACL table doesn't exist.
Today this only works on multi-asic because port-channels use the same ACL as front panel ports and port-channels are included in the global namespace for `acl_table_ports`:
```
for k, v in port_channels[namespace].iteritems():
    acl_table_ports[namespace].append(k)
    acl_table_ports[''].append(k)
```

Reply:
The change you are undoing was made because of this issue https://github.com/Azure/sonic-utilities/issues/2185. This issue is still open, so to avoid issues on the other devices I had suggested on the change.

Reply:
Ah I see, okay fair enough I haven't tried this out on non-Arista devices so I can't confirm if would fail on those devices. I'll do this conditionally on single asic systems as you suggested.